ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -310,11 +327,12 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
 {
     int ret, fi_version;
     struct fi_info *hints;
-    struct fi_info *providers = NULL, *prov = NULL;
+    struct fi_info *providers = NULL, *prov = NULL, *prov_cq_data = NULL ;
```

Comment:
multiple lines make code easier to read :).

Reply:
sure